src/app/components/technical-record/technical-record.component.html

```diff
@@ -430,32 +430,32 @@ <h1 class="govuk-heading-xl title">Technical record</h1>
-                  <td class="govuk-table__cell">{{ isNullOrEmpty(techRecordsJson.techRecord[0].trainGbWeight) ? "-" : techRecordsJson.techRecord[0].trainGbWeight }}</td>
-                  <td class="govuk-table__cell">{{ isNullOrEmpty(techRecordsJson.techRecord[0].trainDesignWeight) ? "-" : techRecordsJson.techRecord[0].trainDesignWeight }}</td>
+                  <td id="test-trainGbWeight" class="govuk-table__cell">{{ isNullOrEmpty(techRecordsJson.techRecord[0].trainGbWeight) ? "-" : techRecordsJson.techRecord[0].trainGbWeight }}</td>
+                  <td id="test-trainDesignWeight" class="govuk-table__cell">{{ isNullOrEmpty(techRecordsJson.techRecord[0].trainDesignWeight) ? "-" : techRecordsJson.techRecord[0].trainDesignWeight }}</td>
                   <td class="govuk-table__cell"></td>
                   <td class="govuk-table__cell"></td>
                   <td class="govuk-table__cell"></td>
                 </tr>
                 <tr class="govuk-table__row" *ngIf="techRecordsJson.techRecord[0].vehicleType == 'hgv'">
                   <th class="govuk-table__cell govuk-!-font-weight-bold">Max train</th>
-                  <td class="govuk-table__cell">{{ isNullOrEmpty(techRecordsJson.techRecord[0].maxTrainGbWeight) ? "-" : techRecordsJson.techRecord[0].maxTrainGbWeight }}</td>
-                  <td class="govuk-table__cell">{{ isNullOrEmpty(techRecordsJson.techRecord[0].maxTrainDesignWeight) ? "-" : techRecordsJson.techRecord[0].maxTrainDesignWeight }}</td>
+                  <td id="test-maxTrainGbWeight" class="govuk-table__cell">{{ isNullOrEmpty(techRecordsJson.techRecord[0].maxTrainGbWeight) ? "-" : techRecordsJson.techRecord[0].maxTrainGbWeight }}</td>
+                  <td id="test-minTrainGbWeight" class="govuk-table__cell">{{ isNullOrEmpty(techRecordsJson.techRecord[0].maxTrainDesignWeight) ? "-" : techRecordsJson.techRecord[0].maxTrainDesignWeight }}</td>
                   <td class="govuk-table__cell"></td>
                   <td class="govuk-table__cell"></td>
                   <td class="govuk-table__cell"></td>
                 </tr>
                 <tr class="govuk-table__row" *ngFor="let axelNo of techRecordsJson.techRecord[0].axles">
                   <th class="govuk-table__cell govuk-!-font-weight-bold">Axle {{ isNullOrEmpty(axelNo.axleNumber) ? "-" : axelNo.axleNumber }}</th>
-                  <td class="govuk-table__cell">{{ isNullOrEmpty(axelNo.weights.gbWeight) ? "-" : axelNo.weights.gbWeight }}</td>
-                  <td class="govuk-table__cell">{{ isNullOrEmpty(axelNo.weights.designWeight) ? "-" : axelNo.weights.designWeight}}</td>
+                  <td [id]="'test-gbWeight-'+axelNo.axleNumber" class="govuk-table__cell">{{ isNullOrEmpty(axelNo.weights.gbWeight) ? "-" : axelNo.weights.gbWeight }}</td>
+                  <td [id]="'test-designWeight-'+axelNo.axleNumber" class="govuk-table__cell">{{ isNullOrEmpty(axelNo.weights.designWeight) ? "-" : axelNo.weights.designWeight}}</td>
```

Comment:
typo replace [id] with id

Reply:
Here it is not a typo. Using [id] instead of id actually allows us to generate dynamic ids in an ngFor, because axelNo.axleNumber is an variable.